os/kernel/pthread/pthread_create.c

```diff
@@ -281,7 +281,6 @@ int pthread_create(FAR pthread_t *thread, FAR const pthread_attr_t *attr, pthrea
 #ifndef CONFIG_SMP
 	if (attr->affinity != 0) {
 		sdbg("ERROR: Non zero affinity value not allowed when CONFIG_SMP is disabled\n");
```

Comment:
Let's change to warning. When SMP is disabled, setting affinity does not cause any issue. Someone can think this is a problem with this error log.